src/scenekit.cs

```diff
@@ -2398,7 +2398,8 @@ interface SCNRenderer : SCNSceneRenderer, SCNTechniqueSupport {
 
 		[NoWatch, NoTV]
 		[Export ("render")]
-		[Availability (Deprecated = Platform.Mac_10_11 | Platform.iOS_9_0)]
+		[Availability (Deprecated = Platform.Mac_10_11)]
+		[Availability (Deprecated = Platform.iOS_9_0)]
```

Comment:
maybe it would be best, when replacing, to use the latest attribute ? i.e.
`[Deprecated (PlatformName.iOS, 9, 0)]`

Reply:
For sure. It just involves more than 20 minutes and 5 vim macros to bang that out.

If you think it's worth it, I can hack this out (in a new PR).

Reply:
yeah, let's split this out in a few different PR, starting the the old, named attributes
and we can split the work across the remaining people not yet on vacation :)